ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.4.25f1
```

Comment:
It probably won't effect anything but can/should this happen in a separate PR? the CI environment variables would also need updating to match this if we do it here.